data/json/npcs/Backgrounds/meteorologist.json

```
@@ -0,0 +1,11 @@
+[
+  {
+    "id": "BGSS_METEOROLOGIST_1",
+    "type": "talk_topic",
+    "dynamic_line": "Before the cataclysm, I was just a regular meteorologist. In recent years, I have worked at meteostation, monitored the weather, state of air, read and wrote reports, made schedules… In general, ordinary boredom. In my free time I was studying electronics and radio engineering, making various devices, some for work, some just for fun. When the first riots began, I gathered all my things that I managed to carry with me and settled in some radiostation. I lived there for a while, doing all sorts of nonsense, until I met you. Then you helped me make a drone, find out what happened to our state, and you persuaded me to go with you. Since then, I have followed you, trusting you with my life.",
```

```suggestion
    "dynamic_line": "Before the cataclysm, I was just a regular meteorologist.  I worked for the local weather station doing data analysis and stuff.  In general, ordinary boredom.  In my free time I was studying electronics and radio engineering, a bit for work but mostly just as a hobby.  When the riots got serious, my neighborhood was looking pretty sketchy.  I gathered all my things that I managed to carry with me and settled in a radio station that seemed abandoned.  I lived there for a while, doing all sorts of nonsense, until I met you.  I guess you know the rest.",
```
Is 'meteostation' meant to be a brand name? It's not an English term.